pkg/strutil/strutil.go

```diff
@@ -84,6 +84,7 @@ func DedupeStrSlice(in []string) []string {
 // ParseCSVMap parses a string like "foo=x,bar=y" into a map
 func ParseCSVMap(s string) (map[string]string, error) {
 	csvR := csv.NewReader(strings.NewReader(s))
+	csvR.LazyQuotes = true
```

Comment:
labels can contains "(double quotes)
If we don't use LazyQuotes, some labels cannot be parsed (like [this](https://github.com/containerd/nerdctl/runs/7534998440?check_suite_focus=true#step:6:483))

I'm not sure we just turn on LazyQutoes option to solve this problem.
But there is no reason to avoid it, so i turn it on.

Reply:
Could you add a comment line to explain that, and add an unit test in `strutil_test.go`

Reply:
In this pr?